### DISCUSSION TITLE
allow to set the width of a field group

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#6559] [Feature Requests]
just like it was with the divider. For instance I like to set it to `fill`:

![image](https://user-images.githubusercontent.com/10562635/123779280-4ce8f780-d8d2-11eb-9904-a4b70a744f26.png)

(it works if you do it manually in `directus_fields`)

**Comment by benhaynes** [comment]
We wanted to tackle this in the future (after MVP), but needed to think through "half" first. They're are a lot of implications with half+grouping.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
